proto/exocore/dogfood/v1/query.proto

```
@@ -2,6 +2,8 @@
 
 package exocore.dogfood.v1;
 
+import "cosmos/base/query/v1beta1/pagination.proto";
```

Buf (1.47.2)

import "cosmos/base/query/v1beta1/pagination.proto": file does not exist

(COMPILE)